test/runtests.jl

```diff
@@ -15,3 +18,4 @@ include("test_deprecated.jl")
 include("test_normest.jl")
 include("test_diag.jl")
 include("test_chainrules.jl")
+include("test_S_kwarg.jl")
```

Comment:
Maybe this file should be under `test/gpu` for consistency with the Nvidia tests that are already there. I’ll see if I can add a buildkite pipeline for AMD GPUs.